CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 
+## [0.4.0] 2020-06-18
```

Comment:
My thinking here was to bump up the minor version since two major dependencies bumped several majors up. It didn't introduce API incompatibilities at least in `azure-operator`.

Reply:
> to bump up the minor version since two major dependencies bumped several majors up.

That would be correct according to semver :)